openassessment/assessment/api/staff.py

```diff
@@ -3,11 +3,14 @@
 """
 import logging
 from django.db import DatabaseError, IntegrityError, transaction
+from django.utils.timezone import now
 from dogapi import dog_stats_api
 
+from submissions import api as submissions_api
+
```

Comment:
That's about how I end up doing it too, good to know I'm not alone in my thinking!